testsuite/integration/microprofile-tck/config/tck-suite.xml

```diff
@@ -26,6 +26,20 @@
                     <exclude name=".*" />
                 </methods>
             </class>
+
+            <!-- TCK and spec dispute: https://github.com/eclipse/microprofile-config/pull/407 -->
```

Comment:
@jmesnil Is this basically that the spec doesn't adequately specify empty value handling, the TCK is testing things not really specified, the smallrye impl does align with the TCK interpretation and we challenge the TCK?

Reply:
The empty value use case is not consistently specified in the spec.
There are tests in the TCK related to this but passing them introduces inconsistencies (e.g. with regards to empty arrays or optional values) that are not covered by the TCK.

The smallrye-config implementation provides a consistent approach for this but it conflicts with the existing TCK tests. We have challenged the TCK and opened https://github.com/eclipse/microprofile-config/pull/407 (superseded by https://github.com/eclipse/microprofile-config/issues/446) to clarify the spec in the next major version.